src/prefect/task_engine.py

```diff
@@ -270,9 +270,18 @@ def begin_run(self):
             return
 
         new_state = Running()
-        state = self.set_state(new_state)
+
         if PREFECT_EXPERIMENTAL_ENABLE_CLIENT_SIDE_TASK_ORCHESTRATION:
             self.task_run.start_time = self.task_run.state.timestamp
+            self.task_run.run_count += 1
```

Comment:
Can you update line above to be:

`self.task_run.start_time = new_state.timestamp`

Reply:
woops yep, on it

Reply:
thanks for catching this 😅